docs/source/deployment/databricks/choosing_databricks_workflow.md

```diff
@@ -0,0 +1,35 @@
+# Choosing the Right Kedro Workflow for Databricks
+
+Databricks offers integration with Kedro through three principal workflows: The Databricks Deployment Workflow, the Databricks Development Workflow, and the Workflow for Developing Kedro projects on Databricks using notebooks. To enhance your development experience, it is crucial to choose the workflow that best fits your project's needs. Let's break down the advantages and use-cases of each workflow to help you make an informed decision.
```

Comment:
```suggestion
Databricks offers integration with Kedro through three principal workflows:

- Use a Databricks workspace to develop a Kedro project.
- Use an IDE, dbx and Databricks Repos to develop a Kedro project.
- Use a Databricks job to deploy a Kedro project.

To enhance your development experience, it is crucial to choose the workflow that best fits your project's needs. Let's break down the advantages and use-cases of each workflow to help you make an informed decision.
```

It would be nice to break these into an ordered list to make them clearer. I also prefer this order for the workflows, this is the order of easiest / most common to hardest / least common. @SajidAlamQB would you mind moving the individual sections into this order on the page also?

@stichbury I was thinking of making the items in the ordered list clickable links to the relevant sections, but this page is so short maybe its not important. What do you think?

Reply:
I think you're right @jmholzer in that the page is really short. In fact, I've removed the links too as I think they're already used in the copy below to direct the reader to the most appropriate page, so they are repetitive at this point on the page. See what you think.